src/app.tsx

```diff
@@ -1,6 +1,7 @@
-declare let window: any;
 import React, { FunctionComponent, useEffect, useState } from "react";
 import "./app.css";
+import "./definition.ts";
```

Comment:
u shouldnt need this, just need to configure ts to pick up def.

I think if you create an @types folder and put your definitions inside, it should work directly

Reply:
I pushed an update. the overall summary is to remove the import, bring the file to @type and add it into the tsconfig. Is that correct implementation ?